game-core/src/main/java/games/strategy/engine/data/changefactory/ChangeFactory.java

```diff
@@ -96,8 +96,7 @@ public static Change removeUnits(final PlayerID player, final Collection<Unit> u
     return new RemoveUnits(player.getUnits(), units);
   }
 
-  public static Change moveUnits(final Territory start, final Territory end, Collection<Unit> units) {
-    units = new ArrayList<>(units);
```

Comment:
The defensive copy here isn't necessary because both the `AddUnits` and `RemoveUnits` constructors already create a defensive copy of this parameter.